django_x509/tests/test_ca.py

```diff
@@ -614,3 +614,10 @@ def test_generate_ca_with_passphrase(self):
         ca.full_clean()
         ca.save()
         self.assertIsInstance(ca.pkey, crypto.PKey)
+
+    def test_generate_ca_with_generalizedTime(self):
+        """ Any date after 2049, should be in generalizedTime"""
+        ca = self._create_ca(validity_end=datetime(2050, 1, 1, 0, 0, 0, 0))
```

Comment:
I think you need to add `validity_start` too (with a future date) just to be super explicit